pulp_rpm/tests/functional/api/test_fips_workflow.py

```diff
@@ -1,963 +1,213 @@
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_122_cdn_redhat_com_content_eus_rhel_server_7_7_7_x86_64_optional_os(self):  # noqa D102
-        url = "https://cdn.redhat.com/content/eus/rhel/server/7/7.7/x86_64/optional/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_123_cdn_redhat_com_content_eus_rhel_server_7_7_7_x86_64_sat_tools_6_6_os(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/eus/rhel/server/7/7.7/x86_64/sat-tools/6.6/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_124_cdn_redhat_com_content_dist_rhel_server_7_7Server_x86_64_sat_capsule_6_7_os(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/sat-capsule/6.7/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_125_cdn_redhat_com_content_dist_rhel_server_6_6Server_x86_64_sat_tools_6_7_os(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/dist/rhel/server/6/6Server/x86_64/sat-tools/6.7/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_126_cdn_redhat_com_content_dist_rhel_server_7_7Server_x86_64_sat_tools_6_7_os(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/sat-tools/6.7/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_127_cdn_redhat_com_content_dist_layered_rhel8_x86_64_sat_tools_6_7_os(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/sat-tools/6.7/os"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_128_cdn_redhat_com_content_dist_rhel8_8_2_x86_64_appstream_kickstart(  # noqa D102
-        self,
-    ):
-        url = "https://cdn.redhat.com/content/dist/rhel8/8.2/x86_64/appstream/kickstart"
-        self._do_test(url)
-
-    @skip_if(bool, "cdn_client_cert", False)
-    def test_129_cdn_redhat_com_content_dist_rhel8_8_2_x86_64_baseos_kickstart(self):  # noqa D102
-        url = "https://cdn.redhat.com/content/dist/rhel8/8.2/x86_64/baseos/kickstart"
-        self._do_test(url)
-
-    def test_130_mirror_centos_org_centos_8_8_BaseOS_x86_64_os(self):  # noqa D102
-        url = "http://mirror.centos.org/centos-8/8/BaseOS/x86_64/os/"
-        self._do_test(url)
-
-    def test_131_mirror_centos_org_centos_8_8_AppStream_x86_64_os(self):  # noqa D102
-        url = "http://mirror.centos.org/centos-8/8/AppStream/x86_64/os/"
-        self._do_test(url)
+# 'export FIPS_WORKFLOW="anything"' to run this suite
+@pytest.mark.skipif(
+    not os.environ.get("FIPS_WORKFLOW", None),
+    "This is a SIX HOUR test suit - run only when you're sure it's what you need",
+)
+@pytest.mark.parallel
+@pytest.mark.parametrize(
```

Comment:
I marked this test parallel since they don't rely on checking artifacts/content units. Can probably reduce the time it takes to run this test if you are confident it won't deadlock. (Not sure if the parallel and parametrize marks work together though).